examples/lua_REPL/Makefile

```diff
@@ -0,0 +1,51 @@
+APPLICATION = lua_repl
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD_WHITELIST := frdm-k64f iotlab-m3 mulle nrf52840dk nucleo-f411re \
+				   stm32f4discovery
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+CFLAGS_OPT = -Og
```

Comment:
why? :)

Reply:
To make it easier to debug the lua implementation and find the exact lines where error appeared. I can remove it.

Reply:
I would either remove it or add a comment on top